2.Views/WebAppPages/Settings/ProfilePage.cs

```diff
@@ -0,0 +1,22 @@
+﻿using SeleniumTest.Core;
+using SeleniumTest.Core.Interfaces;
+using UIElements.Commons;
+using UIElements.Enums;
+using UIElements.Interfaces;
+using UIElements.Web;
+
+namespace Views.WebAppPages;
+
+public class ProfilePage
+{
+    public readonly IGenericWebDriver _driver;
+    public ITypeable FullNameTextField => new TextField("", new Locator(LocatorType.Id, "FullNameInput"), _driver);
+    public ITypeable EmailTextField => new TextField("", new Locator(LocatorType.Id, "EmailInput"), _driver);
+    public IClickable OkButton => new Button("", new Locator(LocatorType.XPath, "c//div[@class='ui-dialog-buttonset']//span[text()='Ok']"), _driver);
+    public IClickable Settings => new Button("", new Locator(LocatorType.XPath, "//div[@id='ctl00_HeaderTopControl1_PanelHeaderButtons']//a[text()='Settings']"), _driver);
```

Comment:
To follow that same naming convention everywhere, you should add what type of element is the Settings element, I guess that is a button, so SettingsButton, or it could be an Option, Link, etc.

Reply:
I actually wanted to ask you about that because I was going to declare it as "SettingsButton" but it's not a button per se. It's an "a" tag. Should I create another ElementType for this (e.g. Link) or could I just use the Button type?

<img width="342" alt="Captura de Pantalla 2023-02-08 a la(s) 09 05 55" src="https://user-images.githubusercontent.com/101685121/217525533-8b298da7-69e0-4077-887b-8ae1f3e59ed8.png">

Reply:
Fixed. I added a "SettingsButton" element to the HomePage where is also the "LogoutButton".